graspologic/plot/plot.py

```diff
@@ -7,8 +7,11 @@
 import pandas as pd
 import seaborn as sns
 from mpl_toolkits.axes_grid1 import make_axes_locatable
-from sklearn.utils import check_array, check_consistent_length
+from sklearn.utils import check_array, check_consistent_length, check_X_y
 from sklearn.preprocessing import Binarizer
+from scipy import linalg
```

Comment:
is this used?

Reply:
it should be in my _plot_ellipse_and_data() but I will double check!

Reply:
it is!